src/tests/tracing/eventpipe/config/name_config_with_pid.csproj

```diff
@@ -5,8 +5,10 @@
     <!-- Tracing tests routinely time out with jitstress and gcstress -->
     <GCStressIncompatible>true</GCStressIncompatible>
     <JitOptimizationSensitive>true</JitOptimizationSensitive>
+    <EnableNativeEventPipe Condition="'$(TestBuildMode)' == 'nativeaot'">true</EnableNativeEventPipe>
```

Comment:
We added this property, `EnableNativeEventPipe ` to allow a [greater degree of control](https://github.com/dotnet/runtime/pull/80382#issue-1526169655) to opt-in the `EventPipe `library. Without `EventSourceSupport`, which is the other way to enable the `EventPipe `library, some managed library code related to `EventSource` will not be present. Not too sure if that is a good idea/feasible now.